### ISSUE TITLE
Thanks for the inspiration!

### ISSUE THREAD

**Issue by svenove** [clowrey/esphome-esp32-2424s012-lvgl-powermeter#1]
Not really an issue, just wanted to say thanks for posting this repo! :)
Based on your example, I made a (consumption) power meter that shows the live consumption of our house (including usage/cost today/this month):
![powermeter](https://github.com/clowrey/esphome-esp32-2424s012-lvgl-powermeter/assets/4579212/b52fa77b-2992-48aa-b289-57188e652e47)

Code: https://github.com/svenove/ESPHome-ESP32-2424S012-LVGL

**Comment by clowrey** [comment]
Awesome! Glad it was useful to you! I don't even have mine reporting real data yet so may use some of your code for those parts 😁

**Comment by clowrey** [comment]
@svenove you can see the new beta documentation and ways to update the text values here :) They used my example as basis for that gauge but did things the more proper way ;) https://deploy-preview-3678--esphome.netlify.app/cookbook/lvgl#semicircle-gauge

**Comment by svenove** [comment]
Thanks, I’ll have a look at it! 🙂
I was particularly interested in this comment:

> Since LVGL only handles integer values on the [meter](https://deploy-preview-3678--esphome.netlify.app/components/lvgl#lvgl-wgt-mtr) scale, but we want a float precision scale we use the same approach as in the examples above to multiply the needle values by 10.

Does that mean that the indicator will stay on the same spot for values 1.0-1.9 kW? I’ll have to look into that and perhaps modify it with a 10-multiplier. 🙂

**Comment by svenove** [comment]
Seems to be quite a few breaking changes in the lvgl by clydebarrow now. Like this one:
https://github.com/clydebarrow/esphome/commit/b2225fd1d8e69caf68e0a6113d9e2ffddbd9b5c6

Is it still his lvgl that is recommended to use?

**Comment by clowrey** [comment]
> Seems to be quite a few breaking changes in the lvgl by clydebarrow now. Like this one: [clydebarrow/esphome@b2225fd](https://github.com/clydebarrow/esphome/commit/b2225fd1d8e69caf68e0a6113d9e2ffddbd9b5c6)
> 
> Is it still his lvgl that is recommended to use?

Yup its still in active development ;) use this, and make sure you are on the latest ESPhome or it wont compile, they just added the ability to reference a specific commit on the most recent version. 
```yaml
external_components:
  - source:
      type: git
      url: https://github.com/clydebarrow/esphome
      ref: fd15094c0860df23d532881df36cfd16c7da1091 #previous commit - wont be needed in the future
    components: [ lvgl ] 
```
I just uploaded my current beta version here, it also has the *10 to increase precision, its definitely necessary for this type of 0-10 gauge. 
https://github.com/clowrey/esphome-esp32-2424s012-lvgl-powermeter/blob/main/esp32-2424s012-lvgl-powermeter-2.yaml

**Comment by clowrey** [comment]
Oh and if you want to see the most recent ESPhome LVGL and display driver changes etc there is discussion about it on the Discord here: https://discord.com/channels/429907082951524364/1119046373543780363

**Comment by svenove** [comment]
Ah, that’s great! I looked at the docs, but couldn’t find any way of “pinning” to a specific commit, just tags/PRs. Guess I wasn’t reading the latest version of the docs then…

Your discord link seems to be to the UniFi-wireless channel…?

**Comment by clowrey** [comment]
Hmm.. for me it seems to go to the right discord, maybe you have to first join ESPhome discord? https://discord.com/invite/KhAMKrd

**Comment by clowrey** [comment]
And yes! it is just out of beta as of yesterday lol.. I asked the same thing on discord and they showed me that it was possible in beta a week ago :)

**Comment by svenove** [comment]
Joining first seems to be it. 👍

**Comment by clowrey** [comment]
But I dont think the official documentation is updated yet to show that, there was a pull request for it I saw though.. 
https://github.com/esphome/esphome/pull/6446
https://github.com/esphome/esphome-docs/pull/3736

Here you can see it will reference commit hash as valid - once this is pushed to main
https://deploy-preview-3736--esphome.netlify.app/components/external_components.html?highlight=commit
